DemoApp/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Please do not manually edit this file, let Android Studio auto generate. (Because I can tell from the timestamp in the first line is supposed to be auto updated by Android Studio).

Reply:
Let's remove the timestamp to avoid confusing because sometime you have to manually edit this file and timestamp get out of sync quickly.